twitter.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="twitter.css">
+    <title>Twitter</title>
+</head>
+<body>
+    <div class="main-body three-column-grid center-wrapper">
+    <header class="nav-bar three-row-grid">
+            <div class="logo">
+                <img class="twitter-logo" src="/web-images/twitter-logo.png" alt="twitterlogo">
+            </div>
+            <nav>
+                <div class="icons">
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/home.png" alt="home">
+                   <a href="#"><h2 class="nav-headers">Home</h2> </a>    
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/hashtag.png" alt="hashtag">
+                    <a href="#"><h2 class="nav-headers">Explore</h2> </a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/notifications.png" alt="notifications">
+                    <a href="#"><h2 class="nav-headers">Notifications</h2></a>
+                </div>
+                <div class="icons two-column-grid__expand-two">
+                    <img class="home-logo" src="/web-images/message.png" alt="message">
+                    <a href="#"><h2 class="nav-headers">Messages</h2></a>
```

Comment:
You could  simplify this:

`<a href="#"><h2 class="nav-headers">Messages</h2></a>`

and not need an h2 tag inside the anchor tag: 

`<a href="#" class="nav-headers-a">Messages</a>` with a css class `.nav-headers-a` that sets the text's style and font size.